packages/types/src/Metadata.ts

```diff
@@ -314,11 +313,10 @@ export class RuntimeModuleMetadata extends Struct {
   }
 
   get storage (): StorageMetadata | undefined {
-    const value = (this.get('storage') as Option<StorageMetadata>).value;
-
-    return value
-      ? value.raw
-      : value;
+    // FIXME This should actually return an Option<StorageMetadata>
```

Comment:
To be 100% correct, here, it should return `Option<StorageMetadata>` instead of `StorageMetadata | undefined`.

I'm pretty sure that this will only be used internally, so I would recommend to put `Option<StorageMetadata>`, and we will write `someMetadataModule.storage.unwrap().functions...` (or if .isSome else...). Just want to double check @jacogr

Reply:
Only internally. So we can break it in whichever way we seem fit.